packages/ui/src/pages/ErrorPage/index.tsx

```diff
@@ -21,19 +21,18 @@ const ErrorPage = ({ title = 'description.not_found', message }: Props) => {
     <div className={styles.wrapper}>
       <NavBar />
       <div className={styles.container}>
-        <ErrorIcon className={styles.icon} />
+        <ErrorIcon />
         <div className={styles.title}>{t(title)}</div>
         {message && <div className={styles.message}>{message}</div>}
-        <div className={styles.placeHolder} />
-        <Button
-          className={styles.backBtn}
-          onClick={() => {
-            navigate(-1);
-          }}
-        >
-          {t('action.back')}
-        </Button>
       </div>
+      <Button
+        className={styles.backBtn}
+        onClick={() => {
+          navigate(-1);
+        }}
+      >
+        {t('action.back')}
```

Comment:
This back button looks different from the one on navbar

Reply:
what do you mean by different? One is the page button one is the icon link.  Do you mean text content?

Reply:
Yes, the back button on navbar has icon and text, and text is only available for desktop.
But the back button here only has text. This is the difference I'm talking about

Reply:
The nav back button is included in the Nav link.  Error page includes both nav bar and a standard back button.